activejob/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Teach initializer to set configs directly on `ActiveJob`
+
+    Previously, configs of the form `config.active_job.NAME` were automatically
+    set on `ActiveJob::Base` if it responds to `NAME=`.
+
+    To support the new `use_big_decimal_serializer` flag, setting configs on
+    `ActiveJob` is supported as well.
+
+    *Adrianna Chang* and *Sam Bostock*
+
```

Comment:
I am unsure of if this needs a CHANGELOG entry, as it doesn't _really_ impact consumers, given it's only changing to support an unreleased feature.

That said, it is possible that someone tracking `main` thinks they have adopted `use_big_decimal_serializer`, when in fact they haven't because the config wasn't being set properly.

Reply:
Yeah I don't think it's needed.

Reply:
Okay, removed 👍